### DISCUSSION TITLE
用的3.9.5.81-v4 为啥注入微信就挂了

### DISCUSSION THREAD

**Discussion by 919954713** [ttttupup/wxhelper#231] [Q&A]
用的wxhelper-3.9.5.81-v4\tool\injector\ConsoleApplication.exe  和 Releases 3.9.5.81-v4  下的wxhelper.dll
命令是ConsoleApplication.exe  -i WeChat.exe -p wxhelper.dll -P 19088
没加 -P 好像就可以返回 dll inject successdll path : w dll path : 388  注入结果：1
加了-P 返回   注入结果：1 然后微信就挂了

**Comment by ttttupup** [comment]
二选一，不能同时使用
